### PR TITLE
Add appsilon logo & contact link

### PR DESCRIPTION
**Pull request by ttnsy**
As required on [this](https://app.clickup.com/t/85zrhuw1d) bench task, this PR add appsilon logo & contact links.

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -16,14 +16,26 @@ ui <- div(
   tags$script(src = "https://cdn.tailwindcss.com"),
   tags$script(src = "main.js"),
   div(
-    class = "mb-2 p-1.5",
-    span(
-      class = "text-lg",
-      "🐝",
+    class = "flex items-center justify-between mr-8 mb-2 p-1.5",
```
Comment (Gotfrid) [inline app.R]: I'd add `ml-10` for consistency with other cards, and also I noticed that margin on the right is off due to collapsing margins in the main content. I don't suggest to fix it now, but `mr-6` looks slightly better imo.

**Review by Gotfrid — APPROVED** [review]
LGMT. Did you contact @rszymanski about this repository? I'd feel better if I knew that he's aware that his dashboard is moved into Appsilon org.

**Comment by rszymanski** [comment]
> LGMT. Did you contact @rszymanski about this repository? I'd feel better if I knew that he's aware that his dashboard is moved into Appsilon org.

@Gotfrid Yes, this was agreed with me.

**Comment by ttnsy** [comment]
thank you @Gotfrid & @rszymanski! :smile_cat:

**Pull request merged**